docs/antrea-network-policy.md

```diff
@@ -256,6 +265,44 @@ spec:
       enableLogging: true
 ```
 
+#### ACNP for default Namespace isolation
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
```

Comment:
This is a clarification question on ACNP CRD. As there is a namespace selector in the spec, what is the significance of namespace field in the metadata of the ACNP CRD? Just trying to compare with K8s NP, where the applied to is based out of namespace field in the metadata.

Reply:
actually there is no namespace field in ACNP metadata as it is cluster scoped; also we are not introducing Namespaces field to ANP which has the namespace field in metadata. did that help clear?

Reply:
Checked the code.. missed the comment that makes the struct non-namespaced.
https://github.com/antrea-io/antrea/blob/e8ca54441b116c944accd3297eb7428234562b37/pkg/legacyapis/security/v1alpha1/types.go#L48